sources/packages/api/src/database/entities/application.model.ts

```diff
@@ -189,6 +189,12 @@ export class Application extends RecordDataModel {
   })
   pirDeniedOtherDesc?: string;
 
+  @Column({
+    name: "relationship_status",
+    nullable: false,
+  })
```

Comment:
line 194 and 196 is `contradicting`. did you mean `nullable: true`

Reply:
it should not be nullable, changed line 196

Reply:
When a student saves as draft you will not be able to save the relationship status. So I believe it should be nullable

Reply:
yes you are right @abschwenker